instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/sqsattributes_test.go

```diff
@@ -26,170 +26,86 @@ import (
 	semconv "go.opentelemetry.io/otel/semconv/v1.17.0"
 )
 
-func TestSQSDeleteMessageBatchInput(t *testing.T) {
-	input := middleware.InitializeInput{
-		Parameters: &sqs.DeleteMessageBatchInput{
-			QueueUrl: aws.String("test-queue-url"),
+func TestSQSAttributeSetter(t *testing.T) {
```

Comment:
While this test refactoring makes sense, it's unrelated to the PR's intent. Splitting it into its own PR would make reviews much faster.

Reply:
ok, makes sense, I refactored it since the shape of the setter changed and this code was updated anyway. I'll move it to another PR